buildbot/print_compile_stats.py

```
@@ -13,7 +13,7 @@
 arr = []
 
 f = open(args.path + "/.ninja_log", "r")
-for l in f.readlines():
+for l in f:
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)